doc/rst/developer/bestPractices/TestSystem.rst

```diff
@@ -433,9 +433,7 @@ associated GitHub issue number in the `#issue-number` format, e.g. `#1`.
 
 The rest of the file is optional and free-form. It can be used over the
 future's lifetime to describe in what way the test isn't working or should be
-working, implementation notes, philosophical arguments, etc.  The one-line
-summaries of all outstanding futures can be viewed by running
-``$CHPL_HOME/util/devel/test/list_futures``.
+working, implementation notes, philosophical arguments, etc.
```

Comment:
``list_futures`` was deleted by c0b57deb585509ed9265708e603ad722aa0134e6 in 2011, which introduced ``futures_stats``.  I can't tell if that script can be used for this purpose.

Reply:
I think just removing that sentence is fine.  `future_stats` isn't a direct replacement for `list_futures`